fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/ExplainLevel.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * ExplainLevel.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2025 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.query.plan.cascades;
+
+/**
+ * Explain level.
+ */
+public final class ExplainLevel {
```

Comment:
Is there a reason this isn't an enum? And then either ordinal position or an explicit field on the enum value would be used in place of the explain level integer?

Reply:
I went back and forth a bunch of times and in the end an integer is better because we do a ton of arithmetic on it (as well as inequalities) and I was just sick of writing `.ordinal()` everywhere.